src/Dashboard/Filters/GroupRequesterFilter.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Dashboard\Filters;
+
+use Change;
+use Group;
+use Problem;
+use Ticket;
+
+class GroupRequesterFilter extends AbstractFilter
+{
+    public static function getName(): string
+    {
+        return _n("Requester group", "Requester groups", 2);
+    }
+
+    public static function getId(): string
+    {
+        return "group_requester";
+    }
+
+    public static function canBeApplied(string $table): bool
+    {
+        /** @var \DBmysql $DB */
+        global $DB;
+
+        return $DB->fieldExists($table, 'groups_id')
+            || in_array($table, [Ticket::getTable(), Change::getTable(), Problem::getTable()]);
```

Comment:
This has been copied from the `GroupTechFilter`. Technically, one may find its logical, but from the end user point of view, I am not sure it is a good idea to filter assets based on the "linked group" when a value is selected in in the "requester group" filter.

Using an asset and being the requester of a ticket are different concept.

@orthagh Your opinion?